src/main/java/com/yelp/nrtsearch/server/luceneserver/search/TerminateAfterWrapper.java

```diff
@@ -20,13 +20,9 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
+
+import org.apache.lucene.search.*;
 import org.apache.lucene.index.LeafReaderContext;
-import org.apache.lucene.search.CollectionTerminatedException;
-import org.apache.lucene.search.Collector;
-import org.apache.lucene.search.CollectorManager;
-import org.apache.lucene.search.LeafCollector;
-import org.apache.lucene.search.Scorable;
-import org.apache.lucene.search.ScoreMode;
```

Comment:
Don't use .* Here. You can change your intellij config to avoid this.

Reply:
Thanks! has configured the IJ and disabled the * import.